projects/kit/src/lib/masks/time/util/tests/stringify-time.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {describe, expect, it} from '@jest/globals';
+import type {MaskitoTimeParams} from '@maskito/kit';
+import {maskitoStringifyTime} from '@maskito/kit';
+
+const params: MaskitoTimeParams = {mode: 'HH:MM:SS.MSS'};
```

Comment:
Also add tests for the such tricky modes like `MM.SS.MSS` and `SS.MSS`, please

Reply:
Done.

Reply:
Done.